packages/components/src/core/InputSearch/index.tsx

```diff
@@ -100,8 +128,9 @@ const InputSearch = forwardRef<HTMLDivElement, InputSearchProps>(
           sdsStyle={sdsStyle}
           sdsStage={hasValue ? "userInput" : "default"}
           onChange={handleChange}
-          onKeyPress={handleKeyPress}
+          onKeyDown={handleKeyPress}
```

Comment:
The onKeyPress event has been deprecated and replaced by the newer onKeyDown event.